charts/rollups-validator-node/README.md

```diff
@@ -0,0 +1,309 @@
+<!--- app-name: Redis&reg; -->
+
+# Package for Cartesi Rollups validator nodes
+
+Refer to the [official documentation](https://docs.cartesi.io/cartesi-rollups/overview/) for more information about Cartesi Rollups.
+
+## TL;DR
+
+```console
+helm install my-release oci://docker.io/cartesi/rollups-validator-node
+```
+
+### OR
+
+```console
+helm install my-release oci://ghcr.io/cartesi/charts/rollups-validator-node
+```
+
+### OR
+
+```console
+helm repo add cartesi https://cartesi.github.io/helm-charts
+helm install my-release cartesi
```

Comment:
I think we need the chart name here:

```suggestion
helm install my-release cartesi/rollups-validator-node
```

Or, instead of using `my-release`, use `my-rollups-validator-node` like we do [here](https://cartesi.github.io/helm-charts/), like that:

```suggestion
helm install my-rollups-validator-node cartesi/rollups-validator-node
```